Classes/BaseAdditionalFieldProvider.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace Sng\Additionalscheduler;
+
+/*
+ * This file is part of the "additional_scheduler" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ */
+
+use \TYPO3\CMS\Scheduler\AdditionalFieldProviderInterface;
+use TYPO3\CMS\Scheduler\Controller\SchedulerModuleController;
+use TYPO3\CMS\Scheduler\Task\AbstractTask;
+
+
+/**
+ * Class BaseAdditionalFieldProvider
+ * @author Marc Munos
+ * @package Sng\Additionalscheduler
```

Comment:
I prefer avoiding name in comments in open source project. All contributors are visible on this repo, is it a problem for you?

Reply:
It's ok. I've just copied what you did for Utils.php.